app/views/unattended/report_templates/host_-_compare_content_hosts_packages.erb

```diff
@@ -17,39 +17,47 @@ template_inputs:
 model: ReportTemplate
```

Comment:
While you're at it, can you add
```
require:
- plugin: katello
  version: 3.16.1
```
to the metadata so it is only seeded when using katello with a version that supports it?